docs/source/user/api.rst

```diff
@@ -476,7 +476,7 @@ Batch user creation
 
 .. code-block:: text
 
-    /api/v1/<organization-slug>/radiusbatch/
+    /api/v1/radiusbatch/add/
```

Comment:
It should be just `/api/v1/radiusbatch/`, posting to a resource implies adding an object as per REST convention